File: Data Structures & Algorithms/two-integer-sum/submission-2.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans;
        vector<vector<int>> dict(nums.size(), vector<int>(2));
        for(int k=0; k<nums.size(); k++)
        {
            dict[k][0] = nums[k];
            dict[k][1] = k;
        }
        sort(dict.begin(), dict.end());

        int i = 0, j = nums.size()-1;
        
        while(i<j)
        {
            if(dict[i][0] + dict[j][0] == target)
            {
                ans.push_back(dict[i][1]);
                ans.push_back(dict[j][1]);
                break;
            }
            else if(dict[i][0] + dict[j][0] < target)
             i++;
            
            else j--;              
        }
        sort(ans.begin(), ans.end());

        return ans;
        
    }
};

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-2.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        arr = [a for a in nums]

        # import heapq
        # heapq.heapify(arr)
        # n = len(arr)
        # for _ in range(n - k):
        #     heapq.heappop(arr)
        arr.sort(reverse=True)
        print(arr)
        
        return arr[k-1] 